nginx/nginx.conf

```diff
@@ -145,6 +145,18 @@ http {
             return ngx.redirect("https://" .. ngx.var.host .. path, ngx.HTTP_MOVED_PERMANENTLY)
           }
         }
+
+        # redirect "/beta-opt-in/?target=" to the target URL
+        location ~ ^/beta-opt-in/?$ {
+            rewrite_by_lua_block {
+                local target = ngx.var.args:match("target=(.*)")
+                if target then
```

Comment:
This is just a safety check right? Or have you seen cases where `target` has no value.

Reply:
It may be overly cautious, but shouldn't hurt to handle.